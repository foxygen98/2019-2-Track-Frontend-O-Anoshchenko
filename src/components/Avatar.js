import React from 'react'
import { ReactComponent as AvatarSvg } from '../assets/buttons/avatar.svg'
import styles from '../styles/Chat.module.css'

function Avatar() {
    return(
        <AvatarSvg className={styles.Avatar} />
    )
}

export default Avatar