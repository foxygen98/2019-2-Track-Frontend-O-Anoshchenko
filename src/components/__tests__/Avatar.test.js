import React from 'react'
import renderer from 'react-test-renderer'
import Avatar from '../Avatar.js'

it('avatar renders correctly', () => {
    const avatar = renderer.create(<Avatar />).toJSON()
    expect(avatar).toMatchSnapshot()
})