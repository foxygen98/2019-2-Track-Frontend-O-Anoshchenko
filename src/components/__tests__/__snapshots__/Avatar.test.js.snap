// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`avatar renders correctly 1`] = `
<svg
  className="Avatar"
>
  avatar.svg
</svg>
`;
